recipes/qt-multimedia/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+
+if test "$CONDA_BUILD_CROSS_COMPILATION" = "1"
+then
+  CMAKE_ARGS="${CMAKE_ARGS} -DQT_HOST_PATH=${BUILD_PREFIX}"
+fi
+
+if [[ $(uname) == "Linux" ]]; then
+  CMAKE_ARGS="${CMAKE_ARGS} -DFEATURE_vulkan=ON"
+fi
+
+cmake -LAH -G "Ninja" ${CMAKE_ARGS} \
+  -DCMAKE_PREFIX_PATH=${PREFIX} \
+  -DCMAKE_FIND_FRAMEWORK=LAST \
+  -DCMAKE_INSTALL_RPATH:STRING=${PREFIX}/lib \
+  -DCMAKE_MESSAGE_LOG_LEVEL=STATUS \
+  -DFEATURE_linux_v4l=OFF \
+  -DQT_DEFAULT_MEDIA_BACKEND=ffmpeg \
+  -B build .
```

Comment:
Does that build folder already exist? If not, can we create it to be clear(er)?

Reply:
Hmm, I took this from https://github.com/conda-forge/qt-charts-feedstock/blob/main/recipe/build.sh, do you just want a `mkdir -p build` before the cmake command?

Reply:
I think that would be a slight improvement. It's better to know where these folders come from (some projects do have a checked-in `build` folder and this can lead to weird effects), so I'm not super keen on how CMake apparently will create the build folder silently.

Personally I prefer the pattern
```
mkdir build
cd build

cmake [options] \
    ..  # look ma, no -B
```
but I guess it's just a matter of habit.